package.json

```diff
@@ -21,13 +21,24 @@
   ],
   "license": "MIT",
   "scripts": {
+    "precommit": "lint-staged",
+    "postcommit": "git update-index --again",
```

Comment:
This is exciting. What does it achieve?

Reply:
@marxian We've found this to help with editors with built-in git clients from reverting linting.

Reply:
:+1: I get it. That's useful knowledge. Thanks.

Reply:
From here, btw: https://www.npmjs.com/package/lint-staged#using-with-jetbrains-ides-webstorm-pycharm-intellij-idea-rubymine-etc